build/verify-tags.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script makes sure each versioned API in service catalog
+#has as json tag
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+result=0
+
+find_files() {
```

Comment:
What files is this finding?

Reply:
a comment might help, but I believe this is finding all of the files in our repo to scan